webgpu/lessons/webgpu-orthographic-projection.md

```diff
@@ -391,8 +391,8 @@ newZ = x * -s + z * c;
 X rotation
 
 <div class="webgpu_center"><pre class="webgpu_math">
-newY = y *  c + z * s;
-newZ = y * -s + z * c;
+newY = y * c + z * -s;
+newZ = y * s + z *  c;
```

Comment:
This does not match the matrix ilustration, but it's correct:

Rotating +Y by 90° gives +Z.
Rotating +Z by 90° gives -Y.